foursq.js

```diff
@@ -0,0 +1,51 @@
+class Place {
+  constructor(name, cat, lat, lng, url, hereNow) {
+    this.name = name;
+    this.cat = cat;
+    this.lat = lat;
+    this.lng = lng;
+    this.url = url;
+    this.hereNow = hereNow;
+  }
+}
+
+// FourSquare API
+function fourSquareFunction(search) {
+  var places = [];
+
+  var radius = 8000; // meters
+
+  // on checkbox click trigger this search!!!
+
+  var url = 'https://api.foursquare.com/v2/venues/search?' +
+    'll=38.894470,-77.036583&' +
+    'client_id=PDKD4SZGV2WUM3HW00FFLWJGUMMLSFLMG4UGKN4DEGDH0GWB&' +
+    'client_secret=JWQ5SO32GUFAKVDX5QWE1PPIHSKER40VTOWT01PUSA1O42TS&' +
+    'v=20170101&' + // time to be changed...
+    'categoryId=' + search + '&' +
+    'radius=' + radius + '&';
+
+  $.ajax({
+    url: url,
+    method: 'GET'
+  }).done(
+    function(data) {
+
+      var venues = data.response.venues;
+
+      for (var i = 0; i < venues.length; i++) {
+
+        var name = venues[i].name;
+        var cat = venues[i].categories[0].name;
```

Comment:
Any reason to limit this to just the first category?

Reply:
When looking at the results of 'venues' in the console, categories is an Array of size 1. This was the only way I could return category. 

Any ideas about this?

Reply:
If this is always an index of 0, then I cannot think of anything better. If this could be larger, may want to consider another for loop with an index of "j" for instance